server/controllers/appController.js

```diff
@@ -0,0 +1,68 @@
+const db = require('../database.js');
+const appController = {};
+
+// Create a new application in the database
+appController.createApp = (req, res, next) => {
+  const company = req.body.company;
+  const position = req.body.position;
+  const contact = req.body.contact;
+  const notes = req.body.notes;
+  const status = req.body.staus;
+  const applied_at = req.body.applied_at;
+  const queryString = `
+      INSERT INTO Applications (company, position, contact, notes, status, applied_at)
+      VALUES ($1, $2, $3, $4, $5, $6);
+      `;
+
+  db.query(queryString, [company, position, contact, notes, status, applied_at])
+    .then(() => {
+      return next();
+    })
+    .catch((err) => {
+      return next({
+        log: `Error occurred with appController.createApp middleware: ${err}`,
+        status: 500,
+        message: { err: 'An error occurred while creating a new application.' },
+      });
+    });
+};
+
+// Retrieve all applications from the database
+appController.getApps = (req, res, next) => {
```

Comment:
We'll want to modify the query to get only those apps that belong to the logged-in user.